docs/overview/getting-started/avalanche-consensus.md

```diff
@@ -2,21 +2,50 @@
-Computers can reach a consensus by following a set of steps called a consensus protocol. Avalanche
+
+Consensus is the task of getting a group of nodes to come to an agreement on a decision.
+Nodes can reach a consensus by following a set of steps called a consensus protocol. Avalanche
 is a new consensus protocol that is scalable, robust, and decentralized. It has low latency and high
 throughput. It is energy efficient and does not require special computer hardware. It performs well
 in adversarial conditions and is resilient to "51% attacks." This document explains the Avalanche
-consensus protocol. The white-paper is [here.](https://www.avalabs.org/white-papers)
+consensus protocol. The white-paper is [here](https://www.avalabs.org/white-papers).
+
+![Consensus Comparison](/img/Consensus-protocol-comparison.png)
+
+Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+[validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
+determining whether a
+[transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
+accepted or rejected, it asks a small, random subset of validators whether they think the
+transaction should be accepted or rejected. If the queried validator thinks the transaction is
+invalid, has already rejected the transaction, or prefers a conflicting transaction, it replies that
+it thinks the transaction should be rejected. Otherwise, it replies that it thinks the transaction
+should be accepted.
+
+If a sufficiently large portion (_alpha_ α) of the validators sampled reply that they think the
+transaction should be accepted, the validator prefers to accept the transaction. That is, when it is
+queried about the transaction in the future, it will reply that it thinks the transaction should be
+accepted. Similarly, the validator will change its preference to prefer another transaction if a
+sufficiently large portion of the validators replies that another conflicting transaction should be
+accepted instead.
+
+The validator repeats this sampling process until _alpha_ of the validators queried reply the same
+way (having the same preferred transaction) for _beta_ β consecutive rounds.
+
+In the common case when a transaction has no conflicts, finalization happens very quickly. When
+conflicts exist, honest validators quickly cluster around conflicting transactions, entering a
+positive feedback loop until all correct validators prefer that transaction. This leads to the
+acceptance of non-conflicting transactions and the rejection of conflicting transactions.
 
+![How Avalanche Consensus Works](/img/howavalancheconsensusworks.png)
 
-## Video
+Avalanche Consensus guarantees (with high probability based on system parameters) that if any honest
+validator accepts or rejects a transaction, all honest validators will accept or reject that
+transaction.
```

Comment:
```suggestion
Avalanche Consensus guarantees that if any honest validator accepts a transaction, then all
honest validators will accept that transaction.
```

I think it's better to make this read easily for newcomers rather than reference high probability and system parameters here as we would expect newcomers to Avalanche to read this page.